spark/src/test/scala/org/apache/comet/exec/CometExecSuite.scala

```diff
@@ -1240,7 +1235,7 @@ class CometExecSuite extends CometTestBase {
         .saveAsTable("bucketed_table2")
 
       withSQLConf(
-        "spark.comet.exec.sortMergeJoin.disabled" -> "true",
+        CometConf.COMET_EXEC_SORT_MERGE_JOIN_ENABLED.key -> "false",
```

Comment:
true -> false change, is it related?

Reply:
This is to disable sort merge join. Previously it is a `disabled` config. Now it is `enabled` config.